service/internal/access/v2/pdp_test.go

```diff
@@ -336,6 +380,231 @@ func (s *PDPTestSuite) SetupTest() {
 		[]string{"cloud"},
 	)
 
+	// Initialize registered resources
+	s.fixtures.classificationRegRes = &policy.RegisteredResource{
+		Name: "classification",
+		Values: []*policy.RegisteredResourceValue{
+			{
+				Value: "topsecret",
+				ActionAttributeValues: []*policy.RegisteredResourceValue_ActionAttributeValue{
+					{
+						Action: testActionRead,
+						AttributeValue: &policy.Value{
+							Fqn:   testClassTopSecretFQN,
+							Value: "topsecret",
+						},
+					},
+				},
+			},
+			{
+				Value: "secret",
+				ActionAttributeValues: []*policy.RegisteredResourceValue_ActionAttributeValue{
+					{
+						Action: testActionRead,
+						AttributeValue: &policy.Value{
+							Fqn:   testClassSecretFQN,
+							Value: "secret",
+						},
+					},
```

Comment:
If our logic is correct, I don't think we should need this `read` action set on `secret, confidential, or public` attributes because it should propagate down hierarchically from `topsecret`